### PR TITLE
ES6ify loadfonts header JS

### PR DESCRIPTION
**Pull request by sndrs**
cc @janua

### CODE REVIEW

File: common/app/common/JsMinifier.scala [hunk 1]
```diff
@@ -37,13 +37,16 @@ object JsMinifier {
...
-
     //Aggressive
     //options.setWarningLevel(DiagnosticGroups.DUPLICATE_VARS, CheckLevel.WARNING)
     //options.setWarningLevel(DiagnosticGroups.MISSING_PROPERTIES, CheckLevel.WARNING)
 
+    options.setLanguageIn(CompilerOptions.LanguageMode.ECMASCRIPT6_STRICT)
+    options.setLanguageOut(CompilerOptions.LanguageMode.ECMASCRIPT3)
+
+    /* Disable some checks */
+    options.setWarningLevel(DiagnosticGroups.LINT_CHECKS, CheckLevel.OFF)
```
Comment (janua) [inline common/app/common/JsMinifier.scala]: Can we just change the one above to `OFF` instead?

Comment (sndrs) [inline common/app/common/JsMinifier.scala]: doh!

**Comment by janua** [comment]
Looks good! :+1: 
Remember it's on in CODE and PROD!

**Comment by rich-nguyen** [comment]
:+1: nearly the weekend, why not eh?

**Pull request merged**